### PR TITLE
init project

### PR DESCRIPTION
**Pull request by HaVuNguyen**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.framgia.music_09"
+        minSdkVersion 19
```
Comment (DoanVanToan) [inline app/build.gradle]: sao lại để minSdk tận 19

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    package="com.framgia.music_09">
+
+    <application
+
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: remove bl

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: remove hardcode

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        android:theme="@style/AppTheme">
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
```
Comment (Tuanlvt) [inline app/src/main/AndroidManifest.xml]: remove this line

File: app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
+    </style>
+
```
Comment (Tuanlvt) [inline app/src/main/res/values/styles.xml]: remove this line

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!"
```
Comment (Tuanlvt) [inline app/src/main/res/layout/activity_main.xml]: hard code

File: app/src/main/res/drawable/ic_launcher_background.xml [hunk 7]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Tuanlvt) [inline app/src/main/res/drawable/ic_launcher_background.xml]: remove it

File: app/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 8]
```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Tuanlvt) [inline app/src/main/res/drawable-v24/ic_launcher_foreground.xml]: remove it

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml [hunk 9]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Tuanlvt) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml]: remove it

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Tuanlvt) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: remove it

File: .idea/misc.xml [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+      </value>
+    </option>
+  </component>
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/build/classes" />
+  </component>
+  <component name="ProjectType">
+    <option name="id" value="Android" />
+  </component>
+</project>
```
Comment (Tuanlvt) [inline .idea/misc.xml]: enter end line

File: app/src/main/res/layout/activity_main.xml [hunk 12]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity"></android.support.constraint.ConstraintLayout>
```
Comment (Tuanlvt) [inline app/src/main/res/layout/activity_main.xml]: change to  />

**Pull request merged**